src/views/Rules/ListRules/index.jsx

```diff
@@ -398,45 +432,45 @@ function ListRules(props) {
     }
   }, [username]);
 
-  useEffect(() => {
-    setProductChannelFilter(
-      searchQueries
-        ? searchQueries.filter(Boolean).join(productChannelSeparator)
-        : ALL
-    );
-  }, [searchQueries]);
-
-  const filteredRulesWithScheduledChanges = useMemo(
-    () =>
-      productChannelFilter === ALL || !searchQueries
-        ? rulesWithScheduledChanges
-        : rulesWithScheduledChanges.filter(rule => {
-            const [productFilter, channelFilter] = searchQueries;
-            const ruleProduct =
-              rule.product ||
-              (rule.scheduledChange && rule.scheduledChange.product);
-            const ruleChannel =
-              rule.channel ||
-              (rule.scheduledChange && rule.scheduledChange.channel);
-
-            if (ruleProduct !== productFilter) {
-              return false;
-            }
-
-            if (channelFilter) {
-              if (ruleChannel.indexOf('*') === -1) {
-                if (ruleChannel !== channelFilter) {
-                  return false;
-                }
-              } else if (!channelFilter.startsWith(ruleChannel.split('*')[0])) {
-                return false;
-              }
-            }
+  const filteredRulesWithScheduledChanges = useMemo(() => {
+    let filteredRules = clone(rulesWithScheduledChanges);
 
-            return true;
-          }),
-    [searchQueries, productChannelFilter, rulesWithScheduledChanges]
-  );
+    // Pending signoff switch
+    if (filteredRules && Boolean(query.onlySignoffs)) {
+      filteredRules = filteredRules.filter(rule => rule.scheduledChange);
```

Comment:
I think we need to do one of two things, either:
a) Change this to filter on rules where the length of `rule.scheduledChange.signoffs` is less than the sum of required signoffs
b) Relabel the toggle/query ary to indicate that we're filtering on scheduled changes, not just things requiring signoff.

Other than that, this patch looks great.